app/views/wpcc/your_details/new.html.erb

```diff
@@ -111,43 +125,46 @@
           </div>
         </div>
         <div class="details__row" data-dough-component="PopupTip">
-          <div class="form__row">
-            <fieldset>
-              <legend class="details__calculate-heading"><%= t('wpcc.details.calculate.legend') %></legend>
-              <p class="details__calculate-intro">
-                <%= t('wpcc.details.calculate.details_html_1') %>
-                <%= render 'wpcc/tooltips/trigger' %>
-                <%= t('wpcc.details.calculate.details_html_2') %>
-              </p>
+         <fieldset>
+            <legend class="details__calculate-heading"><%= t('wpcc.details.calculate.legend') %></legend>
+            <p class="details__calculate-intro">
+              <%= t('wpcc.details.calculate.details_html_1') %>
+              <%= render 'wpcc/tooltips/trigger' %>
+              <%= t('wpcc.details.calculate.details_html_2') %>
+            </p>
 
-              <div data-dough-popup-container 
-                   class="popup-tip__container details__helper details__helper--block">
-                <p data-dough-popup-content class="popup-tip__content">
-                  <span class="details__helper-title--no-js">
-                    <%= t('wpcc.details.calculate.qualifying_earnings_title') %>:
-                  </span>
-                  <%= t('wpcc.details.calculate.qualifying_earnings') %>
-                </p>
-                <%= render 'wpcc/tooltips/close' %>
-              </div>
+            <div data-dough-popup-container
+                 class="popup-tip__container details__helper details__helper--block">
```

Comment:
This spacing looks weird. I'm guessing it was done to keep to the 80 char limit, but sometimes in views I think its okay to overrun the line limit to avoid dangling attributes. However, we have no standards / consensus on this so would be good to get other views, particularly frontenders - @johnplayer1982, @davidtrussler, @thecountgs, @tomas-stefano, @pmcostello @norm @rnalexander